python/MooseDocs/extensions/bibtex.py

```diff
@@ -91,7 +91,6 @@ def postTokenize(self, page, ast):
             if not has_bib:
                 core.Heading(ast, level=2, string='References')
                 BibtexBibliography(ast, bib_style='plain')
-                print('Adding Bibliography:', page.local)
```

Comment:
Was this intentional?

Reply:
This was a change Andrew added into the PR when he was helping me resolve a few problems. He said it was something he meant to remove in a previous PR.